### PR TITLE
Catch unknown data type and default to unspecified.

### PR DESCRIPTION
**Pull request by gabrielhaim**

### CODE REVIEW

File: vertica_python/vertica/column.py [hunk 1]
```diff
@@ -90,7 +90,10 @@ def __init__(self, col):
         self.props = ColumnTuple(col['name'], col['data_type_oid'], None,
                                  col['data_type_size'], None, None, None)
 
-        self.converter = self.DATA_TYPE_CONVERSIONS[col['data_type_oid']][1]
+        try:
+            self.converter = self.DATA_TYPE_CONVERSIONS[col['data_type_oid']][1]
```
Comment (pedercandersen) [inline vertica_python/vertica/column.py]: Could you also do this like:
```
self.converter = self.DATA_TYPE_CONVERSIONS.get(col['data_type_oid'], 0)[1]
```
?

Might be a bit more concise, but other than that, looks good.

Comment (gabrielhaim) [inline vertica_python/vertica/column.py]: DATA_TYPE_CONVERSIONS is a list, you can't use `get` like you would do for a dictionary.

**Comment by twilly** [comment]
LGTM. :shipit:

File: vertica_python/vertica/column.py (hunk 1 above)
Comment (pedercandersen) [inline vertica_python/vertica/column.py]: Ah you're right, IndexError it is then!

**Pull request merged**